app/javascript/profileDropdown/flagButton.js

```diff
@@ -58,10 +40,44 @@ export function initFlag() {
             flagButton.innerHTML = `Flag ${profileUserName}`;
           }
         })
-        .catch((e) => window.alert(`Something went wrong: ${e}`));
+        .catch((e) => {
+          Honeybadger.notify(
+            isUserFlagged ? 'Unable to unflag user' : 'Unable to flag user',
+            userData.profileUserID,
+          );
+          // do we want to alert here, or _only_ log to honeybadger?
```

Comment:
I forgot I had added this comment - this was based on my confusion only skimming the feedback which included "also" pretty clearly. 

I'll remove that  since it will bother me to leave this asking answered questions.